sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>Azure Event Hubs is a highly scalable publish-subscribe service that can ingest millions of events per second and stream them to multiple consumers.  This client library allows for both publishing and consuming events using Azure Event Hubs. For more information about Event Hubs, see https://azure.microsoft.com/en-us/services/event-hubs/</Description>
     <Version>5.0.0-preview.6</Version>
-    <PackageTags>Azure;Event Hubs;EventHubs;.NET;AMQP;IoT</PackageTags>
+    <PackageTags>Azure;Event Hubs;EventHubs;.NET;AMQP;IoT;windowsazureofficial;azureofficial</PackageTags>
```

Comment:
Should we do this on the [Blobs CheckpointStore](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blobs/src/Azure.Messaging.EventHubs.CheckpointStore.Blobs.csproj) project too?

Reply:
Yes. Missed that one. Will add now.

Reply:
Added.